### ISSUE TITLE
Fortnite Crashes Randomly >> EXCEPTION_ACCESS_VIOLATION

### ISSUE THREAD

**Issue by BlauerToad** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#695]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game [Required]

Fortnite Battle Royale

### Game Platform [Required]

- [ ] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [X] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Amazon Games app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

AMD Ryzen 5 2600X

### Graphic Card [Required]

Intel Arc A770 16GB

### GPU Driver Version [Required]

31.0.101.5186

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [ ] Windows 11 23H2
- [ ] Windows 11 22H2
- [ ] Windows 11 21H2
- [X] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[igcit_ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/14154203/igcit_ssu.txt)

### Description and steps to reproduce [Required]

The crash occures randomly. I just open the game via Epic Games Launcher and join a Battle Royale Match. Sometimes the game crashes after 5 Minutes, sometimes it crashes after half an our... The Unreal Engine Crash Reporter will open after the game crashes with no useful information.

I tried playing with DX11 which did not crash, but I did not play that long. Mainly because it was not enjoyable -> stuttery gameplay with bad framerate compared to DX12.

### Game graphic quality [Required]

- [X] Low
- [X] Medium
- [X] High
- [X] Ultra

### Game resolution [Required]

2560x1440

### Game VSync [Required]

Off

### Game display mode [Required]

- [X] Fullscreen
- [ ] Windowed
- [ ] Borderless Windowed

### Detailed game settings [Required]

If you need the settings page in english, just ask for it :)

![Settings_1](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/92584570/07e98db8-f7d9-438c-9192-a1739b43dd20)
![Settings_2](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/92584570/b0c1aad9-b263-4124-a447-6e6d0f865150)
![Settings_3](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/92584570/e1d38bd8-68d8-4289-b4be-f72f3bde3d48)
![Settings_4](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/92584570/7f943ee1-ffe8-4327-a5ff-5f1af60c361b)

### Device / Platform name

Desktop Pc

### Crash dumps [Required, if applicable]

[FortniteGame-backup-2024.02.03-16.27.37.log](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/14154245/FortniteGame-backup-2024.02.03-16.27.37.log)
[UEMinidump.dmp](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/14154226/UEMinidump.dmp)

**Comment by Arturo-Intel** [comment]
@BlauerToad
I was unable to reproduce this behavior, played around 40m with the settings you shared without any trouble

I don't know if you run a DDU cycle when you install the news driver (please try it),
According to the SSU file you shared, I do have twice ram as you, let me retest with 16gb

![image](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/92489452/86bd5861-e5c6-4292-b77e-71a5b79048bb)
Note: you can see when each match started from the fall of FPS (loading screen)  and nop, didn't win any of them :p

**Comment by BlauerToad** [comment]
Hey @Arturo-Intel
thanks for your investigation.

I can understand if you could not reproduce the error. As I said, it really occurs randomly in my case. Sometimes I can finish a round, sometimes the game crashes after 5 minutes...

I recently installed the latest driver (31.0.101.5186) after a DDU run. In my case #687, this has also led to success so far and solved this problem. However, it has not brought any improvement in Fortnite.

In the last screenshot of my settings, there is a setting for GPU debugging. I would activate this option, but I don't know where this data is stored. Maybe that would help you?

In "FortniteGame-backup-2024.02.03-16.27.37.log" the interesting part is at the end, in case you missed it. Of course, I can understand if you overlook the "important" lines in a large log, or if it unfortunately does not provide you with any helpful information.

---
> nop, didn't win any of them :p

Neither do I 😅. In good rounds it will crash of course...

**Comment by BlauerToad** [comment]
Hey,
so I just played another three rounds with advanced gpu debugging but I have not found any additional crash data. I will just paste another log-file here with an slightly different error (kernel32.dll instead of ntdll.dll) and an extra "CrashContext.runtime-xml" file. I added the file extension ".txt" because of upload restrictions.

On line 164 it shows the CallStack which crashed. You can search for `<IsCrashed>true</IsCrashed>`

[FortniteGame-backup-2024.02.06-21.11.33.log](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/14186360/FortniteGame-backup-2024.02.06-21.11.33.log)
[CrashContext.runtime-xml.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/14186361/CrashContext.runtime-xml.txt)

Small remark: I have activated the "Sound visualization" setting and have not yet tested whether this might be the cause.

**Comment by Darklegion1978** [comment]
Good Morning @BlauerToad,

I had a similiar issue like you with Fortnite, it wasnt the driver of the graphics card.
Randomly Fortnite destroys his files after an Update himself.

Can you try this step and test if it works for you too?

Go to C:\Program Files\Epic Games\Fortnite\FortniteGame\Binaries\Win64\ and delete the file in the error "FortniteClient-Win64-Shipping.exe".

Open the Epic Games Launcher and click the Gear icon next to the word Launch.

Select Verify and let it re-download the file.

I hope it will help you as like me.

Greetings Roman

**Comment by BlauerToad** [comment]
Hello @Darklegion1978,

thanks for sharing your experience.

In the meantime, there was a major Fortnite update and a new graphics driver (31.0.101.5194) was released, which I also installed. I can't say yet whether the bug has been fixed, as I've only played one match so far. I will provide an update in the near future.

I had already verified the game files several times, but without deleting the .exe file. I also reinstalled the game once, unfortunately without success. This should have solved the problem if it was due to a corrupted file.

If the problem still occurs, I will try your tip anyway.

As I said, I'll get back to you in the near future after I've played a little longer.

Greetings Chris

**Comment by BlauerToad** [comment]
As promised, an update from me.

As I said, I played a few rounds with driver 31.0.101.5194 and didn't experience any problems, but I never played for long.

Yesterday I installed driver 31.0.101.5330 and played a bit more today. After about an hour and a half I thought the problem was really fixed, but unfortunately my game crashed shortly after. I then started it again and it didn't take 15 minutes before it crashed again...

I also made a CaprameX recording with 31.0.101.5194. It looks very similar to @Arturo-Intel's.

![image](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/92584570/c388a556-0b16-4f46-acad-eaeba9227bd8)

**Comment by Darklegion1978** [comment]
@BlauerToad

I play Fortnite also every day, also with all driver versions. And the last one too.

I have absolutely no problems, not in dx12 and also not in low settings, normal I play just in low but tested it yesterday in dx12 too.

No issues and no crash 🤷

Hope you will get fixed your error soon and if, let me know to play some rounds in FN 😁👍

PS. Im German too 😉

**Comment by BlauerToad** [comment]
Hi guys, 

I have now been able to fix my problem. It seems that Fortnite is heavily utilizing the RAM / memory controller in a specific way. I had initially suspected that the problem might have something to do with the RAM. However, tests such as Memtest86 or TesMem5 did not show any errors. However, after I raised the SoC voltage slightly, the error seems to be fixed.

The case can now be closed. Thank you.
